src/Listener/AddClientAssets.php

```diff
@@ -21,13 +21,13 @@ class AddClientAssets
      */
     public function subscribe(Dispatcher $events)
     {
-        $events->listen(ConfigureWebApp::class, [$this, 'addAssets']);
+        $events->listen(Rendering::class, [$this, 'addAssets']);
     }
 
     /**
      * @param ConfigureClientView $event
```

Comment:
Please update this docblock

Reply:
Sorry, I'm not sure what I should be instead. New to Flarum.
I see this on the namespace changes thread:
```{Event/ConfigureWebApp → Frontend/Event/Rendering}```

Reply:
No worries! Simply change line 28:

```
      * @param ConfigureClientView $events
```

to:

```
      * @param Rendering $events
```

Reply:
Oh duh! Think I got it, thanks.